rakelib/default_plugins.rb

```diff
@@ -119,6 +119,8 @@ module RakeLib
       logstash-input-stdin
       logstash-input-tcp
       logstash-output-stdout
+      logstash-output-null
+      logstash-codec-rubydebug
```

Comment:
not really, good catch.

On Wed, Mar 9, 2016 at 4:36 PM Pier-Hugues Pellerin <
notifications@github.com> wrote:

> In rakelib/default_plugins.rb
> <https://github.com/elastic/logstash/pull/4782#discussion_r55536096>:
>
> > @@ -119,6 +119,8 @@ module RakeLib
> >        logstash-input-stdin
> >        logstash-input-tcp
> >        logstash-output-stdout
> > +      logstash-output-null
> > +      logstash-codec-rubydebug
>
> rubydebug? we need that fore core testing?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/elastic/logstash/pull/4782/files#r55536096>.
>